internal/models.py

```diff
@@ -25,14 +26,14 @@ class Member(models.Model):
         blank=True,
         verbose_name=_("Committees"),
     )
-    role = models.CharField(max_length=64, blank=True, verbose_name=_("Role"))
-    email = models.EmailField(blank=True, null=True, verbose_name=_("Email"))
-    phone_number = PhoneNumberField(max_length=32, default="", blank=True, verbose_name=_("Phone number"))
-    study_program = models.CharField(max_length=32, default="", blank=True, verbose_name=_("Study program"))
+    role = UnlimitedCharField(blank=True, verbose_name=_("Role"))
+    email = models.EmailField(blank=True, verbose_name=_("Email"))
```

Comment:
My interpretation is that `Member.email` is the email for contacting the member, whereas (`Abstract`)`User.email` is for logging in. They're often the same address, but also often not 🙂 I've renamed this specific field to `contact_email` in a soon-to-be-pushed commit, to remove this potential for confusion 😄

Reply:
Ah, I see. I wondered why it couldn't be reused but `contact_email` makes more sense since not everyone likes to be notified through their student mail

Reply:
Yes, true 😄